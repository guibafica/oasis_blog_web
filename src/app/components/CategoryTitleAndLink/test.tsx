import { render, screen } from '@testing-library/react'

import { CategoryTitleAndLink } from '.'

describe('<CategoryTitleAndLink />', () => {
  it('should render the heading', () => {
    const { container } = render(<CategoryTitleAndLink />)

    expect(
      screen.getByRole('heading', { name: /CategoryTitleAndLink/i })
    ).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot()
  })
})
