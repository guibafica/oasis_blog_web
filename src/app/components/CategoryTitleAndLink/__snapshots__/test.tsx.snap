// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CategoryTitleAndLink /> should render the heading 1`] = `
<main>
  <h1>
    CategoryTitleAndLink
  </h1>
</main>
`;
